index.js

```diff
@@ -43,6 +43,79 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Ramda
+//.
+//. Ramda provides a number of partial functions. These necessitate the use
```

Comment:
I completely agree with the sentiment, @Avaq. I'm also very much in favour of precise definitions. There are two problems with using the phrase "functions which might return `null` or `undefined`" to describe partial functions:

  - there are total functions which may return these values (e.g. [`S.I`][1]); and
  - there are partial functions which never return these values (e.g. [`R.divide`][2]).

I will make changes based on your various suggestions, Aldwin. I greatly appreciate your input!


[1]: https://sanctuary.js.org/#I
[2]: http://ramdajs.com/docs/#divide